locales/locales.php

```diff
@@ -16,6 +16,7 @@ class GP_Locale {
 	public $google_code = null;
 	public $preferred_sans_serif_font_family = null;
 	public $facebook_locale = null;
+	public $alphabet = null;
```

Comment:
Shouldn't we make Latin the default? Similar to `ltr` for `$text_direction`.

```suggestion
	public $alphabet = 'latin';
```

Reply:
Done!